vmdk_plugin/main.go

```diff
@@ -97,8 +98,8 @@ func main() {
 	var driver volume.Driver
 
 	// Define command line options
-	driverName := flag.String("driver", "vmdk", "Volume driver")
-	logLevel := flag.String("log_level", "debug", "Logging Level")
+	driverName := flag.String("driver", vmdkDriver, "Volume driver")
```

Comment:
I'll try out having one listener run as a goroutine and check that its working ok.

BUt do we need this capability at all?